index.html

```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Document</title>
+  </head>
```

Comment:
- [ ] [OPTIONAL] I see this  `<head/>` part you miss to create `link` for `styles.css` file therefore header's color is not working.  Please implement it!
![image](https://user-images.githubusercontent.com/63915024/170098362-fcefc086-4b68-4639-9304-15213e1e6dae.png)